kostal/README.md

```diff
@@ -1,14 +1,21 @@
 # KOSTAL
 
+VERSION = "1.3.1"
+
 # Requirements
 
-This plugin is designed to retrieve data from a [KOSTAL](http://www.kostal-solar-electric.com/) inverter module (e.g. PICO inverters).
+This plugin is designed to retrieve data from a [KOSTAL](http://www.kostal-solar-electric.com/) inverter module (e.g. PIKO inverters)
+Since UI-version 6 json-format is supported
+
+Tested with Kostal Piko 3.0 (single phase)  UI-Version 06.20
+dc-line 2, ac line 2 and ac line 3 values unchecked/untested
```

Comment:
I would kike to see the list of possible values in the configuration section. And we should mentioned that the "old" PIKO converters are still supported with the html format, new ones with the JSON format.